server_action_navigate/__init__.py

```diff
@@ -0,0 +1,8 @@
+from . import models
```

Comment:
please move this too `hooks.py` and leave here only `from .hooks import uninstall_hook`

Reply:
done

Reply:
@simahawk moved everything from init.py to hooks.py but 'from . import models'
Because pre-commit removes the above line from hooks.py. So I kept it in init.py. Is there any other way?

Reply:
bad phrasing, sorry: import models should stay here, of course :)

```
from . import models
from .hooks import uninstall_hook
```

Reply:
ok, then it's done